test/wordFormatter.spec.js

```diff
@@ -0,0 +1,24 @@
+import {WordFormatter} from '../src/wordFormatter'
+import {GuessRegister} from '../src/guessRegister'
+import {Rules} from '../src/rules'
+
+let wordFormatter
+let guessRegister
+let rules
+
+beforeEach(() => {
+  rules = new Rules("hello")
```

Comment:
I guess what is happening, what do you think to have something a little more obvious like

`rule.secretWordIs("hello")` or something like that